tests/client/amo/testApi.js

```diff
@@ -216,9 +211,9 @@ describe('amo.api', () => {
         .returns(Promise.resolve({ results: [] }));
 
       return getLatestUserReview()
-        .then(unexpectedSuccess, (error) => {
-          assert.match(error.message, /user and addon must be specified/);
-        });
+        .then(unexpectedSuccess, (error) =>
```

Comment:
I agree, I would really like to ditch implicit returns everywhere actually. The rule is annoying about it 😒